content/docs/getting-started/create-admin-group/index.md

```diff
@@ -33,3 +39,49 @@ Roadie admins have special powers such as the ability to add and remove Backstag
 ![A link that says "Log in to GitHub"](./sidebar-log-into-github.png)
 
 ⚠️  &nbsp;It can take some time for Roadie to refresh the list of teams from GitHub teams. If you do not see admin functions immediately, please wait a few minutes and try again.
+
+
+### Manual admin group configuration with group entities
```

Comment:
I think I'd add "(advanced alternative)" or something after this. To indicate that it's an optional step.